src/expressServer.js

```diff
@@ -18,11 +18,9 @@ const { readFileSync } = require('fs');
 const { join } = require('path');
 const expressStaticGzip = require('express-static-gzip');
 const recommendationRoutes = require('./routes/recommendationRoutes');
+const imgur = require('./views/imgur/imgur');
```

Comment:
Here this is importing a javascript file from your `views` folder, `imgur.js`. That javascript file seems to be being used to define Express routes. If that's the case, we should have that file in our `routes` folder for easier discoverability and to make it clear to other developers what the functionality of that file is. We can also name it `imgurRoutes` to follow the pattern of other files in that folder.

While I like the idea of co-locating all the code that has to do with `imgur` in to a shared folder; one thing that makes me lean towards following the existing approach of having just `.ejs` files in the `views` folder and just routes files in the `routes` folder is that the `imgur` upload capability could potentially in the future be used for other features apart from the user profile. Like what if we want admins to be able to upload custom album art or something like that.

Reply:
ImgurRoutes.js created and relocation of files done , commit 749d1e07510f8835388f0e1f8989f97383a7be79, commit 39723399eaf86ce4f2c099e07370da4d1e1cf18d